zerver/tests/test_outgoing_webhook_interfaces.py

```diff
@@ -238,6 +228,17 @@ def make_response(text: str) -> requests.Response:
         )
         service_handler = self.handler
 
+        # verify we don't try to send a message with empty text/content
+        response = make_response(text=json.dumps(dict(text="")))
+
+        with mock.patch("zerver.lib.outgoing_webhook.send_response_message") as m:
+            process_success_response(
+                event=event,
+                service_handler=service_handler,
+                response=response,
+            )
+        self.assertFalse(m.called)
+
```

Comment:
I'm not a big fan of tests that only contain failures, just because it can sometimes end up being the case that the test setup becomes invalid after some refactoring and the tests all fail for the wrong reason, which in turn allows bugs to creep in.  So we prefer a test that ends with something that is a success with small changes in the thing being attempted up to that point.

While you're touching this, can you try to rework this one to fit that pattern?